index.js

```diff
@@ -17,9 +18,12 @@ module.exports = function (opts) {
 			return;
 		}
 
-		(new Vulcanize(opts)).process(file.path, function (err, inlinedHtml) {
+		// vulcanize expects target path to be relative to abspath
+		let filePath = opts.abspath ? path.relative(opts.abspath, file.path) : file.path;
```

Comment:
I've never used an `fs` API that requires a relative filepath. This should clearly be handled internally by the module. Yes, it would be easy to fix here, but I'm not interested in having lots of workarounds for Vulcanize, when it could simply be fixed there instead. Vulcanize should pretty much have this line in their code.

> and make the changes there, but wouldn't it be better to fix it in general?

Yes, and that's why I'm asking you to open this PR on the Vulcanize repo. Otherwise every consumer of Vulcanize will need to do this workaround, instead of just Vulcanize fixing it.